docs/API.md

```diff
@@ -107,17 +107,23 @@ A `<Link>` can know when the route it links to is active and automatically apply
 
 #### Props
 ##### `to`
-The path to link to, e.g. `/users/123`.
-
-##### `query`
+Can be either a string or an object.
```

Comment:
Can you caveat that the below is only true for a "standard" history? Otherwise it's just a mostly opaque location descriptor.